js/graph_config.js

```diff
@@ -910,19 +907,25 @@ GraphConfig.load = function(config) {
             };
         }
     };
-    
+
     /**
      * Get an array of suggested graph configurations will be usable for the fields available in the given flightlog.
-     * 
+     *
      * Supply an array of strings `graphNames` to only fetch the graph with the given names.
      */
     GraphConfig.getExampleGraphConfigs = function(flightLog, graphNames) {
+
+        console.log("Running getExampleGraphConfigs");
+
         var
             result = [],
             i, j;
 
+        var sysConfig = flightLog.getSysConfig();
+        var debugModeName = DEBUG_MODE[sysConfig.debug_mode];
```

Comment:
```suggestion
        const sysConfig = flightLog.getSysConfig();
        const debugModeName = DEBUG_MODE[sysConfig.debug_mode];
```

Reply:
This change was from a commit that was accidentally added in this PR by mistake.  

It was intended to add the debug name into the text that appears to the right of the graph, but it didn't work, so I've removed that commit.

That removes the need to make this change, but I'll put it into the code I was testing.

Thank you again.